README.md

```diff
@@ -4,6 +4,12 @@ Cost Monitoring Construct is a CDK library that helps monitor costs for AWS clou
 
 With the Cost Monitoring Construct, you can share the responsibility of cost management between developers and business holders. This is achieved through the creation of meaningful reports that enable the business team to make informed decisions. Additionally, the Construct generates boilerplate code that can be used to apply these decisions in practice, making it easier to stay on top of your budget.
 
+# How to use Cost Monitoring Construct?
+
+To use Cost Monitoring Construct, all you need is to have the AWS CDK installed and set up. Once you have that, you can install the package from the repository of your choice.
+
+For more information on using this Construct in TypeScript checkout the [TypeScript documentation](docs/typescript.md).
```

Comment:
Now please also link to samples here.

Reply:
This is done @shahinism. Would you please approve it if you thinks is good for now?